app/scripts/filters/resources.js

```diff
@@ -349,18 +349,22 @@ angular.module('openshiftConsole')
         return url;
     };
   })
-  .filter('routeLabel', function(routeHostFilter, routeWebURLFilter, isWebRouteFilter) {
+  .filter('routeWebURLNoPath', function(routeHostFilter){
+    return function(route, host){
+        var scheme = (route.spec.tls && route.spec.tls.tlsTerminationType !== "") ? "https" : "http";
+        var url = scheme + "://" + (host || routeHostFilter(route));
+        return url;
+    };
+  })
+  .filter('routeLabel', function(routeHostFilter, routeWebURLNoPathFilter, isWebRouteFilter) {
     return function(route, host) {
       if (isWebRouteFilter(route)) {
-        return routeWebURLFilter(route, host);
+        return routeWebURLNoPathFilter(route, host);
```

Comment:
this is going to affect everywhere that uses routeLabel today, I don't think that is what we want.  In this context we are showing you the path separately, so you don't need to see it, but in other places we don't and we may want to see the path included.